src/pages/peer-review-board-page/PeerReviewBoardPage.tsx

```diff
@@ -53,11 +63,32 @@ const generateCardList = (cardList: PeerReviewBoardPage_user) =>
   cardList.map((user: UserType) => <UserCard user={user} key={user.id} />);
 
 export default function PeerReviewBoardPage(props: Props) {
+  const classes = useStyles(props);
+
+  const { state } = useLocation<LocationState>();
+  const history = useHistory<LocationState>();
+  const open = state?.showDialog ?? false;
+  const showFab = state?.done ?? false;
```

Comment:
```suggestion
  const showFab = allDown && !showDialog
```

Reply:
I dont think `showFab` is dependent on `showDialog`, it is only dependent on state of done.
for instance, we can show the dialog while having fab, then if user clicks the first button, we go to game page, otherwise the dialog will be closed, with fab on the page.

Reply:
مشکلی نیست پس این مدلی می‌شه:

```
const showFab = allDone
```